extensions/amp-story/1.0/amp-story-scroll.css

```diff
@@ -14,16 +14,15 @@
  * limitations under the License.
  */
 
-
 amp-story[scroll] {
-  contain: none !important;
+  overflow-y: auto !important;
 }
 
 [scroll] amp-story-page {
   position: relative !important;
   height: 100vh !important;
 }
 
-amp-story[scroll] > amp-story-page.i-amphtml-layout-container[distance] {
+[scroll] amp-story-page[distance] {
```

Comment:
Why the drop in specificity?  Did something change?

Reply:
Yup sorry for the lack of context: #17189

Reply:
is it possible to add more contents in amp-stories? and is it possible to scroll under theirs?

Reply:
Each page if an amp-story is not scrollable.  We are considering adding the ability to swipe up a page to reveal more content, tracked by #15845.

This issue tracks an experimental UI treatment that uses scrolling to transition between pages.

Reply:
thank you @newmuis